model/src/commonMain/kotlin/Contributor.kt

```diff
@@ -0,0 +1,9 @@
+package io.github.droidkaigi.confsched2019.model
+
+data class Contributor(
+    val id: Int?,
```

Comment:
I don't want to use nullable type for Model when it is not needed.
If we do it, we have to check nullable type in the whole App. 😭